docs/Brew-Test-Bot-For-Core-Contributors.md

```diff
@@ -1,23 +1,26 @@
-# Brew Test Bot for Maintainers
+# BrewTestBot for Maintainers
 
-If a build has run and passed on `brew test-bot` then it can be used to quickly bottle formulae.
+`brew test-bot` is the command our CI runs to test and build bottles for formulae.
 
-## Bottling
+## Publishing Bottles
 
-If a pull request is correct and doesn't need any modifications to commit messages or otherwise:
+If CI is passing on a pull request and it doesn't need any modifications (i.e. commit message, revision bump, etc.):
 
 1. Review and approve the pull request. Be sure to thank the contributor!
-2. Wait for BrewTestBot to automatically merge the pull request. This happens about once an hour. BrewTestBot will comment if there is a failure.
+2. Wait for BrewTestBot to automatically merge the pull request. This job usually starts within a minute if both of the following are true:
+    - The pull request is approved by a maintainer who has write access to homebrew-core.
+    - CI is passing, and all jobs are green.
```

Comment:
Does “CI is passing” not imply that all jobs are green?

Reply:
Yeah, that makes sense.